src/software/er_force_simulator_main.cpp

```diff
@@ -30,6 +31,9 @@ int main(int argc, char **argv)
     desc.add_options()("division",
                        boost::program_options::value<std::string>(&args.division),
                        "div_a or div_b");
+    desc.add_options()("realism_config",
+                       boost::program_options::bool_switch(&args.enable_realism),
+                       "realism simulator");  // install terminal flag
```

Comment:
Could you update the arg name and description here as well? The name `realism_config` should only be used to refer to the actual config. Any variable used for enabling/disabling the realism feature should ideally be named consistently (`enable_realism`)